addons/xbmc.python/addon.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<addon id="xbmc.python" version="2.24.0" provider-name="Team-Kodi">
-  <backwards-compatibility abi="2.1.0"/>
+<addon id="xbmc.python" version="2.25.0" provider-name="Team-Kodi">
+  <backwards-compatibility abi="2.25.0"/>
```

Comment:
I was referring to a compat layer for our python interface in general, while first use case would be RenderCapture stuff. But from your last comment I get that simmulating the old API of RenderCapture could not be handled self-contained in a compatibility layer by mocking stuff - so that won't help here then. Sorry for the noise.